jobs-builder/jobs/dependencies.yaml

```diff
@@ -0,0 +1,66 @@
+# This file contain the configurations to generate Kernel jobs.
+---
+- job-template:
+    name: 2.x-kernel-{flavor}-{arch}
+
+    # Template default variables
+    #
+    arch: x86_64
+    os: ubuntu1804
+
+    project-type: freestyle
+    description: Build Kernel(s) for Kata Containers usage
+    disabled: false
+    concurrent: true
+    logrotate:
+      daysToKeep: 60
+      numToKeep: 8
+    node: !include-jinja2: include/os2node.yaml.inc
+    scm:
+      - git:
+          url: https://github.com/kata-containers/kata-containers
+          branches:
+            - 'refs/heads/2.0-dev'
+    triggers:
+      - timed: 'H 0-23/6 * * 1-5'
+    wrappers:
+      - ansicolor:
+          colormap: "xterm"
+      - openstack:
+          single-use: True
+      - timestamps
+      - timeout:
+          timeout: 5
```

Comment:
Is this in seconds? Why 5 ooi?

Reply:
Increased to 180.

Reply:
I just realized the timeout in the yaml is in minutes but the job builder will convert to seconds.